src/capi/azext_capi/helpers/kubectl.py

```diff
@@ -0,0 +1,204 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+This module contains helper functions for the az capi extension.
+"""
+
+# pylint: disable=missing-docstring
```

Comment:
We should write [docstrings](https://peps.python.org/pep-0257/) instead of disabling `pylint`. They don't have to be detailed.